packages/store-api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -151,7 +151,7 @@ export const validateCart = async (
       orderNumber: updatedOrderForm.orderFormId,
       acceptedOffer: updatedOrderForm.items.map((item) => ({
         ...item,
-        product: skuLoader.load(item.id),
+        product: skuLoader.load([{ key: 'id', value: item.id }]), // TODO: add channel
```

Comment:
Add the channel here?

Reply:
Yes! when supporting sales channel on orderForm